src/resource/event/event.service.ts

```diff
@@ -33,13 +33,11 @@ export class EventService {
         private readonly exceptionRepository: Repository<EventExceptionEntity>,
     ) {}
 
-    async fetchById(eventId: number): Promise<ReturnEventDTO> {
-        const event = await this.eventRepository.findOne({
+    async fetchById(eventId: number): Promise<ReturnEventDTO | null> {
```

Comment:
The thing is that we don't need every time to have return type of a method explicitly. If you hover at function and see the correct type from a implicit typing, then it is all right. It should be done in the whole app.

Reply:
Done